guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

```diff
@@ -1257,11 +1257,13 @@
 00:54:14.287 --> 00:54:24.415
 [J Jay Couey]: And that insistence is also a false basis for this viral contagion idea and hiding this basic transfection and transformation.
 
-00:54:24.455 --> 00:54:35.483
+00:54:24.455 --> 00:54:37.449
 [J Jay Couey]: So, instead of speaking for an hour, I already probably spoke too long, I want to be able to answer as many questions as anybody wants to throw at me, even from previous talks.
+
+00:54:37.920 --> 00:54:38.417
 [Charles Kovess]: Wonderful.

```

Comment:
I tweaked the timestamps here based on the waveform. Not perfect as it's overlapping speech.

Reply:
I see. I didn't realize you could edit the time stamps like that.

Reply:
Yeah, you can tweak it by hand but I used Subtitle Edit because I can see the waveform of the audio against the time.

Reply:
Ah, OK.

Reply:
Yup basically add a `HH:MM:SS.mmm --> HH:MM:SS.mmm` line above the subtitle that is missing a timestamp. A blank line indicates that it's a new set of captions.